docs/manually-deploying-atmosphere-ansible.md

```diff
@@ -155,15 +142,15 @@ The latest code (usually `master`) for Atmosphere Ansible.
 		60%
 		Networking  10.0.1.27   Tiny1   Atmo Cloud - USA
 		```
-		
+
 	1. Generate hosts
 
 		```
 		cat instances.txt | grep 10| awk '{ i++;printf "vm-%s ansible_host=%s ansible_port=22\n",i,$2}'
 		```
-		
-1. Run Ansible
+
+1. Run Ansible. from `<path-to-repo-clone>/ansible/`:
 
 	```
 	ansible-playbook all-the-pbs.yml -i hosts -e @deploy_vars.yml --limit "vm-*"
-	```
\ No newline at end of file
+	```
```

Comment:
This markdown has come in quite handy for me in the past. So I'm happy to see the improvements - and even happier that is exists. 

:tad